app/models/maintenance_tasks/task.rb

```diff
@@ -200,6 +201,20 @@ def after_error(*filter_list, &block)
         set_callback(:error, :after, *filter_list, &block)
       end
 
+      # Rescue listed exceptions during an iteration and report them to the error reporter, then
+      # continue iteration.
+      #
+      # @param exceptions list of exceptions to rescue and report
+      def report_on(*exceptions)
+        rescue_from(*exceptions) do |exception|
+          if Rails.gem_version >= Gem::Version.new("7.1")
+            Rails.error.report(exception, source: "maintenance-tasks")
```

Comment:
```suggestion
            Rails.error.report(exception, source: "maintenance_tasks")
```
Let's use the gem name I guess? There's not much guidance on the source parameter but I think it makes more sense to use the exact name of the gem.